src/lang/en-US.ts

```diff
@@ -600,6 +601,18 @@ const StorageMappings: ResolvedIntlConfig['messages'] = {
     'storageMappings.bucket.description': `The name of the bucket you have setup to store data in.`,
     'storageMappings.lastUpdated.label': `Last Updated`,
     'storageMappings.message': `Below are all the ${CommonMessages['terms.storageMapping']} that you have read or admin access to. These are the locations that your data is stored. You currently cannot edit these in the UI. If you need an update please ${CTAs['cta.support']}.`,
+
+    'storageMappings.cta.addStorageMapping': `Configure Storage`,
+    'storageMappings.dialog.generate.header': `Configure Storage`,
```

Comment:
If these are going to be the same content please just use a single key

Reply:
I was waiting for Dave to review the content before consolidating content strings. It is looking like this PR will be merged before the content is _officially_ approved, so I do not mind doing so at this time.